src/Form/Type/BuildingType.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Form\Type;
+
+use App\Entity\Building;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class BuildingType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('address')
+            ->add('startTime')
+            ->add('endTime')
```

Comment:
you could add some types here as well, to enforce validations and rendering

Reply:
Types are not mandatory as I have configured options to use entity types.... but sometimes are useful especially for formatting. I'll add. Most of the time default values are ok.